common/utils.go

```diff
@@ -37,11 +36,7 @@ func FormatValue(val any) any {
 	case *time.Time:
 		return val.UTC()
 	case []uint8:
-		s := string(val)
-		if parsed, err := time.Parse(time.DateTime, s); err == nil {
-			return parsed.UTC().Format(time.RFC3339)
-		}
-		return s
+		return string(val)
```

Comment:
I realised that I could just force the mysql connection to always parse the time instead of me doing it. You can see it here:

https://github.com/conduitio-labs/conduit-connector-mysql/blob/f554fd6741499801275b4e9aaf2649d7d708ca4a/source.go#L66-L68

Now, that's actually on the unsafe snapshot pr, I guess I did a git rebase mistake :S

When I started developing this connector I did have plenty of errors with datetimes, and I believe that I fixed it by adding the `parseTime=true` query parameter in the test dsn. I later realized that that's a mandatory query parameter, so I did force it later on, instead of relying in the user.

We might want to remove the parseTime from the test dsn, so that we rely on the implementation.